app/scripts/migrations/153.ts

```diff
@@ -32,9 +32,7 @@ function transformState(
   state: Record<string, unknown>,
 ): Record<string, unknown> {
   if (!hasProperty(state, 'TokensController')) {
-    global.sentry?.captureException?.(
-      new Error(`Migration ${version}: TokensController not found.`),
-    );
+    console.warn(`newState.TokensController is not present`);
```

Comment:
Even though this was not the cause for the Sentry issue; i think it makes sense to downgrade to console.warn